src/rebar3_sbom_cyclonedx.erl

```diff
@@ -39,3 +53,32 @@ uuid() ->
 
 hex(Bin) ->
     string:lowercase(<< <<Hex>> || <<Nibble:4>> <= Bin, Hex <- integer_to_list(Nibble,16) >>).
+
+get_version(File) ->
```

Comment:
Nice addition! I suppose if someone wants incrementing version number in a CI/CD environment they could cache the previous version and use that to derive the version number in future runs.

Just wondering: the spec says the version number should be increased when the SBoM "is changed". So should we also parse the previous XML file and check whether it is in any way different from the current one before bumping the number?

Reply:
I have fixed this and added a new command line option to allow users to skip the modification check.